docs/modules/release-notes/pages/5-5-0.adoc

```diff
@@ -34,17 +38,17 @@ There are no breaking changes in this release.
 * Add JobBuilder to JetService 
 * Introduce a common parent POM for hazelcast and hazelcast-enterprise
 * Improve float array serialization performance
+* Add ZGC bean names to support GC stats https://github.com/hazelcast/hazelcast/pull/26329[#26329]
```

Comment:
```suggestion
* Add ZGC bean names to support GC stats https://github.com/hazelcast/hazelcast/pull/26329[#26329]
* Add CP direct-to-leader operation sending for `ADVANCED_CP` licensed clients
```
Not sure if we mention the license requirement, but we should definitely mention the new direct-to-leader CP operation routing that is available to 5.5+ clients.

Part of this work also provided stability improvements to the CP subsystem in general (by improving member-to-member leadership awareness too), so that might be worth mentioning - we did resolve a known CP issue that caused OOMs due to leadership races, so that could be listed under bugs fixed too if desired.